README.md

```diff
@@ -44,12 +44,27 @@ This image is running with the latest Apache version in the [official PHP reposi
 For the database, you can use and link any SQL server related to MySQL.
 
 Currently if you do not have any MySQL server, the most simple way to run this container is:
+```bash
+# create a network for containers to communicate
+$ docker network create prestashop-net
+# launch mysql container
+$ docker run -ti --name some-mysql --network prestashop-net -e MYSQL_ROOT_PASSWORD=admin -p 3307:3306 -d mysql
```

Comment:
We should probably advise for the tag `5.6` to be installed (or `5.7` for PrestaShop 1.7), to make sure all PrestaShop versions can work properly without additional configuration.